doc/4-instrument/multipcm.md

```diff
@@ -0,0 +1,37 @@
+# MultiPCM instrument editor
+
+the MultiPCM instrument editor contains three tabs: Sample, MultiPCM and Macros.
+
+## Sample
+
+for sample settings, see [the Sample instrument editor](sample.md).
+
+## MultiPCM
+
+MultiPCM contains an ADSR envelope, not unlike Yamaha OPN/OPM envelpes, and simple LFO, also similiar to that found in OPN.
+you may use this tab to set up ES5506-specific parameters:
+
+- **AR**: sets attack rate.
+- **D1R**: sets Decay 1 rate.
+- **DL**: sets Decay Level (analogue of Sustain Level on OPN chips).
+- **D2R**: sets Decay 2 rate (a.k.a. SR, Sustain Rate).
+- **RR**: sets release rate.
+- **LFO rate**: sets speed of LFO.
+- **PM Depth**: sets depth of LFO vibrato.
+- **AM Depth**: sets depth of LFO tremolo/amplitude modulation.
+- **Damp**: enforce quickly fading out the sample over 11 ms.
+- **Pseudo Reverb**: enables reverb-like effect.
+- **LFO Reset**: disables and resets LFO.
+- **Level Direct**: this register is propietary and should be set to ON.
```

Comment:
I think there was a discussion on the meaning of Level Direct.
It disables interpolation between volume changes.

Reply:
Just to be clear: if Level Direct is set to "on", volume change interpolation is enabled or disabled?

Reply:
Disabled.
I'll consider renaming this checkbox.